cmd/gdsutil/main.go

```diff
@@ -149,35 +162,95 @@ func main() {
 			Action:   ldbList,
 			Before:   openLevelDB,
 			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:   "d, db",
-					Usage:  "dsn to connect to trisa directory storage",
-					EnvVar: "GDS_DATABASE_URL",
+				&cli.StringFlag{
+					Name:    "d",
+					Aliases: []string{"db"},
+					Usage:   "dsn to connect to trisa directory storage",
+					EnvVars: []string{"GDS_DATABASE_URL"},
 				},
-				cli.StringFlag{
-					Name:  "o, out",
-					Usage: "path to write CSV data out to",
-					Value: "directory.csv",
+				&cli.StringFlag{
+					Name:    "o",
+					Aliases: []string{"out"},
+					Usage:   "path to write CSV data out to",
+					Value:   "directory.csv",
 				},
 			},
 		},
+		{
+			Name:     "profile:list",
+			Usage:    "list all known GDS profiles",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   listProfiles,
+		},
+		{
+			Name:     "profile:get",
+			Usage:    "get the current GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   getProfile,
+		},
+		{
+			Name:     "profile:set",
+			Usage:    "set the current GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   setProfile,
+		},
+		{
+			Name:     "profile:delete",
+			Usage:    "delete a GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   deleteProfile,
+		},
+		{
+			Name:     "var:list",
+			Usage:    "list the variables in the current GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   listVariables,
+		},
+		{
+			Name:     "var:get",
+			Usage:    "print a variable in the current GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   getProfileVariable,
+		},
+		{
+			Name:     "var:set",
+			Usage:    "set a variable in the current GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   setProfileVariable,
+		},
+		{
+			Name:     "var:delete",
+			Usage:    "delete a variable from the current GDS profile",
+			Category: "profile",
+			Before:   openProfileConfig,
+			Action:   deleteProfileVariable,
+		},
 		{
 			Name:     "peers:add",
 			Usage:    "add peers to the network by pid",
 			Category: "replica",
 			Before:   initReplicaClient,
 			Action:   addPeers,
 			Flags: []cli.Flag{
-				cli.StringFlag{
-					Name:   "u, replica-endpoint",
-					Usage:  "the url to connect the directory replica client",
-					Value:  "replica.vaspdirectory.net:443",
-					EnvVar: "TRISA_DIRECTORY_REPLICA_URL",
+				&cli.StringFlag{
+					Name:    "u",
+					Aliases: []string{"replica-endpoint"},
+					Usage:   "the url to connect the directory replica client",
+					Value:   "replica.vaspdirectory.net:443",
```

Comment:
If the Value is set here, it will be difficult to tell if the user specified it on the command line or if the value should come from the profile; I removed it from the `gds` CLI.

Reply:
Note that we should still remove the Value here in favor of the profiles.